### PR TITLE
OSDOCS-4524: Last minute clean up to 4.12 RNs

### PR DESCRIPTION
**Pull request by opayne1**
[OSDOCS-4524](https://issues.redhat.com//browse/OSDOCS-4524): Last minute clean up to 4.12 RNs

Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4524

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -1266,21 +1259,13 @@ With each major, minor, or patch version release of {product-title}, the HyperSh
...
-
-In a situation where you need disaster recovery for a hosted cluster, you can recover the hosted cluster to the same region within AWS. For more information, see xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc[Disaster recovery for a hosted cluster within an AWS region].
-
 [id="ocp-4-12-rhv"]
 === Red Hat Virtualization (RHV)
 
-* With this update, the oVirt CSI driver logging was revised with new error messages to improve the clarity and readability of the logs.
-* Previously, you had to restart the cluster API provider when the credentials changed. With this release, the cluster API provider automatically updates oVirt and Red Hat Virtualization (RHV) credentials when they are changed in {product-title}.
+This release provides several updates to Red Hat Virtualization (RHV). With this release
+
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:1266]: extra space

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -1947,6 +1883,10 @@ sourceStrategy:
 
 * Previously, incomplete YAML was not synced was occasionally displayed to users. With this update, synced YAML always displays. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084453[*BZ#2084453*])
 
+* Previously, when installing an Operator that required a custom resource (CR) to be created for use, the *Create resource* button could fail to install the CR because it was pointing to the incorrect namespace. With this update, the *Create resource* button works as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2094502[*BZ#2094502*])
+
+* Previously, the *Cluster update* modal was not displaying errors properly. As a result, the *Cluster update* modal did not display or explain errors when they occurred. With this update, the *Cluster update* modal was updated to correctly display errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096350[*BZ#2096350*])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:1888]:
```suggestion
* Previously, the *Cluster update* modal was not displaying errors properly. As a result, the *Cluster update* modal did not display or explain errors when they occurred. With this update, the *Cluster update* modal correctly display errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2096350[*BZ#2096350*])
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -2062,7 +1997,7 @@ sourceStrategy:
 
 * Previously, the File Integrity Operator did not properly handle modifying alerts during an upgrade. As a result, alerts did not include the namespace in which the Operator was installed. With this release, the Operator includes the namespace it was installed into in the alert, making it easier to narrow down what component needs attention. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2112394[*BZ#2112394*])
 
-* Previously, service account ownership for the File Integrity Operator regressed due to underlying OLM updates, and updates from 0.1.24 to 0.1.29 were broken. With this update, the Operator should default to upgrading to 0.1.30. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2109153[*BZ#2109153*]
+* Previously, service account ownership for the File Integrity Operator regressed due to underlying OLM updates, and updates from 0.1.24 to 0.1.29 were broken. With this update, the Operator should default to upgrading to 0.1.30. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2109153[*BZ#2109153*])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-12-release-notes.adoc:2000]:
```suggestion
* Previously, service account ownership for the File Integrity Operator regressed due to underlying OLM updates, and updates from 0.1.24 to 0.1.29 were broken. With this update, the Operator defaults to upgrading to 0.1.30. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2109153[*BZ#2109153*])
```

Comment (JoeAldinger) [reply release_notes/ocp-4-12-release-notes.adoc:2000]: Not sure if that can "should" can be omitted or not. Trying to be more direct.

**Comment by opayne1** [comment]
closing because opening a new PR.

**Pull request closed without merging**